src/main/kotlin/defineGlobalViews.kt

```diff
@@ -10,7 +10,7 @@ fun defineGlobalViews(model: Model, views: ViewSet) {
   views.createSystemLandscapeView("system-overview", "All systems").apply {
     addAllSoftwareSystems()
 
-    val noisySignOnSystems = listOf(HMPPSAuth.system, MoJSignOn.system)
+    val noisySignOnSystems = listOf(HMPPSAuth.system, MoJSignOn.system, AzureADTenantJusticeUK.system)
```

Comment:
_not a blocker_
📝 At this point, it may be worth adding properties/categorisation to these soon.